pkg/kubelet/dockershim/docker_container.go

```diff
@@ -183,7 +183,24 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeApi
 
 // StartContainer starts the container.
 func (ds *dockerService) StartContainer(containerID string) error {
-	return ds.client.StartContainer(containerID)
+	err := ds.client.StartContainer(containerID)
+	if err != nil {
+		return fmt.Errorf("failed to start container %q: %v", containerID, err)
+	}
+
+	info, err := ds.client.InspectContainer(containerID)
```

Comment:
Hm...if we can't inspect the container and/or create symlinks, should we clean up the container? Maybe add a comment here so we can revisit it later.